packages/camera/camera_windows/windows/test/camera_test.cpp

```diff
@@ -497,17 +641,12 @@ TEST(Camera, OnVideoRecordSucceededInvokesCameraChannelEvent) {
   // and second is camera closing message.
   EXPECT_CALL(*binary_messenger, Send(Eq(camera_channel), _, _, _)).Times(2);
 
-  RecordSettings record_settings;
-  record_settings.record_audio = false;
-  record_settings.fps = 5;
-  record_settings.video_bitrate = 200000;
-  record_settings.audio_bitrate = 32000;
```

Comment:
The tests where I removed these extra setters just weren't using the values; there's not reason to set them in every test.